test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,138 @@
+"use strict";
+
+require('./mockmobile');
+var fs = require('fs');
+var path = require('path');
+var crypto = require('crypto');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
+
+
+function deleteFolder(fileLocation, file, cb) {
```

Comment:
Instead of creating this function wouldn't it be easier to use something like https://www.npmjs.com/package/fs-extra-promise? You can add it as a dev dependency in the package.json and just call remove? That function, btw, will handle cases where there is more than one file in the directory.